src/MarketingLayout/MarketingLayout.js

```diff
@@ -58,6 +60,7 @@ class MarketingLayout extends React.PureComponent {
 
   static defaultProps = {
     size: SIZES.small,
+    alignItems: ALIGNMENTS.center,
```

Comment:
When it comes to `static propTypes` and `static defaultProps`, we must use explicit values for the AutoDocs parser.

```suggestion
    alignItems: 'center',
```